graphics/macgui/macwindow.cpp

```diff
@@ -273,11 +270,21 @@ void MacWindow::drawBorder() {
 		titleHeight = _macBorder.getOffset().top - titleY - _macBorder.getOffset().titleBottom;
 		sidesWidth = _macBorder.getOffset().left + _macBorder.getOffset().right;
 	} else {
-		drawSimpleBorder(g);
-		titleColor = _wm->_colorBlack;
-		titleY = 0;
-		titleHeight = _borderWidth;
-		sidesWidth = _borderWidth * 2;
+		warning("MacWindow: No Border Loaded");
```

Comment:
We need to have better handling for this now. Currently, it will crash if no borders were loaded. Thus, I recommend to issue this warning and then call `setBorderType(0xff)` which will load the standard thin border.